dask_image/ndmeasure/_utils/_find_objects.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import pandas as pd
+from dask.delayed import delayed, Delayed
+import dask.dataframe as dd
+
+
+def _array_chunk_location(block_id, chunks):
+    """Pixel coordinate of top left corner of the array chunk."""
+    array_location = []
+    for idx, chunk in zip(block_id, chunks):
+        array_location.append(sum(chunk[:idx]))
+    return tuple(array_location)
+
+
+@delayed
+def _find_bounding_boxes(x, array_location):
+    """An alternative to scipy.ndi.find_objects"""
+    unique_vals = np.unique(x)
+    unique_vals = unique_vals[unique_vals != 0]
```

Comment:
We could potentially do that, as long as we kept track of the mapping between the old and new label integers.

Whether it's faster & worth it would depend on results from some performance testing. I'm inclined to get an implementation in, and then tinker with speed improvements (and anyone who'd like to jump in and try stuff is more than welcome!)

Reply:
Sounds good :)

Reply:
Issue made to track any future performance testing of `find_objects` https://github.com/dask/dask-image/issues/252